cfme/fixtures/v2v_fixtures.py

```diff
@@ -140,7 +145,34 @@ def create_tags(appliance, transformation_method):
     return tag1, tag2
 
 
-def set_conversion_instance_for_rhev(appliance, transformation_method, rhev_hosts):
+def get_vddk_url(provider):
```

Comment:
**Optional** @sshveta  you can also directly retrieve `v6_5_2_v2v` and use it with every version of vmware provider, that should work.